app/collins/models/LshwHelper.scala

```diff
@@ -86,6 +92,41 @@ object LshwHelper extends CommonHelper[LshwRepresentation] {
     )
   }
 
+  protected def reconstructGpu(meta: Map[Int, Seq[MetaWrapper]]): FilteredSeq[Gpu] = {
```

Comment:
i assume this is copy+paste from another reconstruct function? its really hard to read :/